practice1.rb

```diff
@@ -0,0 +1,8 @@
+def solution(num)
+  num_arr = num.to_s.chars.map(&:to_i) # turns num to arr
+  num_arr.sort!.reverse!
+
+  return num_arr.join.to_i
```

Comment:
`return` should be omitted if possible
```suggestion
  num_arr.join.to_i
```